detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/Application.kt

```diff
@@ -32,10 +32,7 @@ internal interface Application {
         override fun run(args: Array<String>) {
             streams.request
                     .subscribeOn(scheduler)
-                    .parallel()
```

Comment:
Actually you can keep this code, it's not going to slow down non-multiplex scenario noticeably and will support multiplex in future automatically

Reply:
Yep, this is more of a paranoid safeguard than a real deal.